data/duke.txt

```diff
@@ -0,0 +1 @@
+T | 0 | sleep
```

Comment:
Are both duke.txt and tasks.txt file being used? If not, you might want to delete one of them.